src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -19,7 +19,7 @@ const useStyles = makeStyles(theme => ({
         height: '100%',
         padding: '3.25rem 0',
         position: 'relative',
-        [theme.breakpoints.down('sm')]: {
+        [theme.breakpoints.down('lg')]: {
```

Comment:
Do we need the breakpoint?

Reply:
I double-checked, and found an issue. "projects" grid is broken..

Reply:
![Screenshot 2022-04-19 085206](https://user-images.githubusercontent.com/2625371/163943332-543660a6-d322-4d7f-9b74-0d2203ad8e19.png)